Linkbus/Linkbus/RouteController.swift

```diff
@@ -249,8 +249,12 @@ extension RouteController {
         //print(apiBusSchedule.routes?.count)
         //let busSchedule = BusSchedule(msg: apiBusSchedule.msg!, attention: apiBusSchedule.attention!, routes: apiBusSchedule.routes!)
         
-        refreshedLbBusSchedule.msg = csbsjuApiResponse.msg!
-        refreshedLbBusSchedule.attention = csbsjuApiResponse.attention!
+        if(csbsjuApiResponse.msg != nil){
+            refreshedLbBusSchedule.msg = csbsjuApiResponse.msg!
+        }
```

Comment:
Adding 

else { refreshedLbBusSchedule.msg = "" }

Doubt it matters though

Reply:
I don't think it matters. Might do that by default?